source/extensions/common/wasm/wavm/BUILD

```diff
@@ -20,6 +20,7 @@ envoy_cc_library(
         '-DWASM_API=""',
         '-DWASTPARSE_API=""',
         '-DIR_API=""',
+        '-DWAVM_API=""',
```

Comment:
FYI the WAVM_API definition supercedes all the other *_API definitions, so you should be able to remove them.

Reply:
Updated. Also removed the missing imports check below.  Thanx!